features/validate_image.feature

```diff
@@ -306,6 +306,31 @@ Feature: evaluate enterprise contract
     }
     """
 
+  # Demonstrate that a validation with no failures, warnings, or successes constitutes a failure as nothing was actually evaluated.
+  Scenario: no failures, warnings, or successes
+    Given a key pair named "known"
+    Given an image named "acceptance/ec-happy-day"
+    Given a valid image signature of "acceptance/ec-happy-day" image signed by the "known" key
+    Given a valid Rekor entry for image signature of "acceptance/ec-happy-day"
+    Given a valid attestation of "acceptance/ec-happy-day" signed by the "known" key
+    Given a valid Rekor entry for attestation of "acceptance/ec-happy-day"
+    Given a git repository named "happy-day-policy" with
+      | main.rego | examples/allow_all.rego |
```

Comment:
Does it still pass if this is changed to happy_day.rego?

Reply:
No.